Project.toml

```diff
@@ -12,16 +11,23 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 
 [compat]
-Blosc = "0.7.1"
 Compat = "3.1.0"
 HDF5_jll = "~1.10.5, ~1.12.0"
 Requires = "1.0"
 julia = "1.3"
 
 [extras]
+Blosc = "a74b3585-a348-5f62-a45c-50e91977d574"
```

Comment:
```suggestion
```

Do we still need Blosc in the main HDF5.jl source?

Reply:
I thought it might be part of the tests still, but perhaps not. Let's omit it and see what happens.